sdk/servicebus/azure-servicebus/azure/servicebus/_servicebus_client.py

```diff
@@ -74,15 +86,40 @@ class ServiceBusClient(object):
 
     """
 
-    def __init__(self, fully_qualified_namespace, credential, **kwargs):
-        # type: (str, Union[TokenCredential, AzureSasCredential, AzureNamedKeyCredential], Any) -> None
+    def __init__(
+        self,
+        fully_qualified_namespace: str,
+        credential: Union[
+            "TokenCredential", "AzureSasCredential", "AzureNamedKeyCredential"
+        ],
+        *,
+        http_proxy: Optional[Dict[str, Any]] = None,
+        user_agent: Optional[str] = None,
+        logging_enable: Optional[bool] = False,
+        transport_type: Optional[TransportType] = TransportType.Amqp,
```

Comment:
This should be `azure.servicebus.TransportType.Amqp`, not a uamqp one to avoid uamqp type leaking.

Reply:
Are we sure we want to promote all these to keyword-only args?
We could never do this for pipeline configuration options in an HTTP-based client....